synfig-studio/src/gui/app.cpp

```diff
@@ -1389,7 +1399,9 @@ DEFINE_ACTION("keyframe-properties", _("Properties"));
 	ACCEL("<Control><Shift>less",    "<Actions>/canvasview/seek-begin");
 	ACCEL("<Control><Shift>greater", "<Actions>/canvasview/seek-end");
 	ACCEL("<Mod1>o",                 "<Actions>/canvasview/toggle-onion-skin");
-	ACCEL("0",                       "<Actions>/canvasview/canvas-zoom-fit");
+	ACCEL("<Control>equal",			 "<Actions>/canvasview/canvas-zoom-in-2"					);
+	ACCEL("<Control>minus",			 "<Actions>/canvasview/canvas-zoom-out-2"					);	
+	ACCEL("<Control>0",              "<Actions>/canvasview/canvas-zoom-fit-2");
```

Comment:
I mean use full spaces like this
![Screenshot from 2020-11-17 13-13-05](https://user-images.githubusercontent.com/24214509/99360533-bcc3e800-28d6-11eb-989b-d3477cb45bcd.png)
Two are still in tabs. Also, please remove extra spaces/tabs before closing brackets for `canvas-zoom-in-2` and `canvas-zoom-out-2`

Reply:
You can see the code looks fine on our IDE but not the same for GitHub. Using space character will fix that :)

Reply:
@Keyikedalube Working On it :)

Reply:
@Keyikedalube Finally made all the required changes.

Reply:
See the screencast please. You still used tab whitespace again...

Reply:
@Keyikedalube Okay. :)

Reply:
@Keyikedalube I removed every tab i used and replaced it with spaces as you shown in the screencast. btw which IDE do you use, i mean its great you can see where it is tab and where it is space. Kindly review it. I also squashed it.

Thanks :)

Reply:
QtCreator. You press `Ctrl + e Ctrl + v` in combination to turn on and off whitespace visualization.

Reply:
Resolved :)